sys-fs/fuse/files/fuse-3.13.1-r1-fix-config.patch

```diff
@@ -0,0 +1,256 @@
+From d7560cc9916b086bfe5d86459cc9f04033edd904 Mon Sep 17 00:00:00 2001
```

Comment:
Thanks. For completeness, can you cinlude a link to the upstream commit at the top of the patch?

Reply:
So this time I remembered to add the commit there, but I did not add it as a link. Fixed now.